boards/arm/cc1352r1_launchxl/cc1352r1_launchxl.dts

```diff
@@ -59,10 +59,25 @@
 			label = "Push button 2";
 		};
 	};
+
+	idle-states {
+		idle: idle {
+			compatible = "zephyr,idle-state";
+			idle-state-name = "runtime-idle";
+			min-residency-us = <1000>;
```

Comment:
Please document either in the DTS or in the commit comment where these times came from.

Reply:
@pabigot these are the current value used for this board look:
 https://github.com/zephyrproject-rtos/zephyr/blob/master/boards/arm/cc1352r1_launchxl/Kconfig.defconfig

The purpose of this pr is move this information from Kconfig to DTS and not review all the values, this can be done later.

Reply:
> The purpose of this pr is move this information from Kconfig to DTS and not review all the values, this can be done later.

If it's being moved from Kconfig to DTS, where are the changes that remove it from Kconfig and update the uses?

Reply:
look above:
> Next step is replace code getting this info from Kconfig. To to this is necessary to replace
the current power states with new ones.

Reply:
Then put it in this PR, or delay this until there's something that uses it.

Reply:
> Then put it in this PR, or delay this until there's something that uses it.

yes, of course this can be done this way, but you know how it goes, someone will come and say 'oh, this is too much for one PR, split it...'. I have no preference, we need to move forward with this though.

Reply:
If it's properly organized, splitting it is trivial.  But seeing only one third of a complete change makes it very hard to see what's going on.

Reply:
I don't believe there's any contention about the missed fixes to the bindings, so those could be pulled out and merged quickly; then changing how residency durations are provided and used would be a nicely self-contained PR. (I suspect; can't tell without seeing it.)